tests/era5_download/test_era5_download.py

```diff
@@ -85,35 +81,19 @@ def test_config_parser_missing_field(base_config, field):
         config_parser(base_config)
 
 
-# --- Invalid date
+# --- Invalid datetime
 @pytest.mark.parametrize(
-    ("date_field", "invalid_date"),
+    ("datetime_field", "invalid_datetime"),
     [
-        ("start_date", "2019-02-31"),
-        ("start_date", "2019-13-01"),
-        ("start_date", "not-a-date"),
+        ("start_datetime", "2019-02-31"),
+        ("start_datetime", "2019-13-01"),
+        ("start_datetime", "2019-01-01T25"),
```

Comment:
Does this last one fail? `("start_datetime", "2019-01-01T25")`

Reply:
Yes, because it's asking for the 25th hour of that date